src/uct/ib/rdmacm/Makefile.am

```diff
@@ -15,16 +15,10 @@ libuct_rdmacm_la_LIBADD   = $(RDMACM_LIBS) $(top_builddir)/src/ucs/libucs.la \
                             $(top_builddir)/src/uct/ib/libuct_ib.la
 libuct_rdmacm_la_LDFLAGS  = $(IBVERBS_LDFLAGS) $(RDMACM_LDFLAGS) -version-info $(SOVERSION)
 
-noinst_HEADERS = \
-	rdmacm_md.h \
-	rdmacm_iface.h \
-	rdmacm_ep.h \
-	rdmacm_def.h
+noinst_HEADERS =
 
 libuct_rdmacm_la_SOURCES = \
-	rdmacm_md.c \
-	rdmacm_iface.c \
-	rdmacm_ep.c
+	rdmacm_component.c
```

Comment:
the whole rdma_cm component should be compiled-in only if have HAVE_RDMACM_QP_LESS